itests/hive-unit/src/test/java/org/apache/hive/beeline/TestBeeLineWithArgs.java

```diff
@@ -665,7 +721,7 @@ public void testEmbeddedBeelineConnection() throws Throwable{
     // Set to non-zk lock manager to avoid trying to connect to zookeeper
     final String SCRIPT_TEXT =
         "set hive.lock.manager=org.apache.hadoop.hive.ql.lockmgr.EmbeddedLockManager;\n" +
-        "create table ${DUMMY_TBL} (d int);\nshow tables;\n";
+        "create table ${DUMMY_TBL} (d int);\nshow tables;\n drop table ${DUMMY_TBL};\n";
```

Comment:
there is create table in

> testTabInScriptFile 
> testBeelineConnectEnvVar
> testBeelineReconnect

as well which might require the drop statement.